README.md

```diff
@@ -0,0 +1,36 @@
+# Spike Detector Fuse for NEST
+This module monitors the spiking activity of the network, and throws a python exception if the spiking activity goes
+beyond the given threshold for the given length of time.
+
+It is a module for the [NEST simulator](http://www.nest-simulator.org).
+
+# Installation
+With the appropriate `nest-config` in your PATH:
+1. First create a directory for the build, say `cmake-build`
+2. `cd cmake-build && cmake ..`
+3. `make && make install`
+4. Run the line `export LD_LIBRARY_PATH=/path/to/nest/installation/lib/nest/:$LD_LIBRARY_PATH` or add it to your
```

Comment:
Regarding the installation, there is already a script that does the installation called `install.sh` in the root directory, I would rather they run that than the above because I have added some compilation flags in that. so the first three points should be replaced by just a single instruction to run the `install.sh` script. Also, make the 4. point just a prerequisite instruction similar to requiring `nest-config` in the PATH.